app/src/main/res/values/strings.xml

```diff
@@ -205,6 +205,10 @@
   <string name="patientsummary_bangladesh_national_id">ID</string>
 
   <string name="patientsummary_contact_doctor">Doctor</string>
+  <plurals name="patientsummary_contact_doctor_patient_info_bps">
```

Comment:
I have tried marking it as untranslatable but for some reason AS didn't allow me to 🤔

Reply:
I tried adding the `translatable=false` attribute on the plural and it seems to work 🤔 

```xml
<plurals name="patientsummary_contact_doctor_patient_info_bps" translatable="false">
    <item quantity="one">*Last %s BP reading*:</item>
    <item quantity="other">*Last %s BP readings*:</item>
</plurals>
```

Reply:
Weird, it's not showing up for me. I will just copy-paste this.

Reply:
Oh, I didn't get the auto-suggestion either. Plurals are a second class citizen from the tooling perspective since they don't show up even in the translation editor. But the attribute exists.